modules/checkbox/react/lib/Checkbox.tsx

```diff
@@ -44,7 +48,7 @@ const CheckboxContainer = styled('div')({
  * This stops the ripple from showing when you hover on the label.
  */
 const CheckboxInputWrapper = styled('div')<Pick<CheckboxProps, 'disabled'>>(
-  {
+  DirectionProvider(direction, {
```

Comment:
If `direction` came from a React context, how would this API change?

Reply:
I don't think this API would change, because we would get the `direction` as a prop and use it here.